tracing/src/test/java/com/palantir/tracing/TraceTest.java

```diff
@@ -21,19 +21,20 @@
 
 import com.palantir.tracing.api.OpenSpan;
 import com.palantir.tracing.api.SpanType;
+import java.util.Optional;
 import org.junit.Test;
 
 public final class TraceTest {
     private static final String ORIGINATING_SPAN_ID = "originating span id";
 
     @Test
     public void constructTrace_emptyTraceId() {
-        assertThatThrownBy(() -> Trace.of(false, "")).isInstanceOf(IllegalArgumentException.class);
+        assertThatThrownBy(() -> Trace.of(false, "", Optional.empty())).isInstanceOf(IllegalArgumentException.class);
```

Comment:
lots of churn haha

Reply:
Agreed, I felt safer adding another param to all the test cases since it means we can't forget to pass the requestId along in production code.